### ISSUE TITLE
CRDAO Spec (V1)

### ISSUE THREAD

**Issue by burrrata** [crdao/dao#1]
# Community Rewards DAO: Specification, Application Process, and Governance settings

**This document is a draft, suggestions on how it can be improved are welcome**

## Philosophy

The Aragon Network is a platform for DAOs, a tool kit for mechanism designers, a framework for decentralised applications, a global jurisdiction and much more. To achieve it's long term goals, it must Experiment with novel incentive mechanisms, grow it's community and foster participation from the wider blockchain ecosystem.

The Community Rewards DAO (CRDAO) will champion and incentivise community participation with a rewards program based on the [1Hive model](https://1hive.org/contribute/allocations-payroll).

## The Community Rewards DAO

- Members of the CRDAO have a non-transferable membership token enabling them to vote in both a mainnet and rinkeby DAO

- Funds awarded to the CRDAO are held in the Mainnet DAO

- Until the Dot Voting application is live on mainnet, dot votes will be held on the rinkeby DAO. The results of these votes will then be paid out manually via the mainnet DAO

- CRDAO members will form an Aragon Working Group to discuss cool things happening in the community. This will either be a keybase group or a thread on the Aragon forum.

- Anyone from within the community can nominate another community member to be rewarded

- CRDAO members (or a CRDAO manager) will add nominations to the CRDAO Address Book and create a weekly dot vote that includes all eligible nominees

- when [SourceCred](https://sourcecred.io) is ready for [Aragon DAO integration](https://devpost.com/software/credao) CRDAO will transition from having members that vote on community nominations to programmatically minting tokens for contributions based on a pagerank style algorithm. This will be based on things like time spent on the forum, likes received, replies and comments, etc... (most likely this will be around the time of the next AGP)

### SourceCred

Cred is a score which represents how much one has contributed to the project. Credits can be calculated based on GitHub, Forum, Rocket Chat metrics or a composite of each. The cred score will fluctuate in both directions; a contributor’s cred might increase (because their contributions have been adding a lot of value) or might decrease (because their contributions are going stale, or being re-assessed downwards). Cred is not transferable between identities

@lkngtn @jvluso and @clarson made great progress with the app at ETH Boston hackathon, you can find a post detailing what they achieved [here](https://devpost.com/software/credao)

### Membership for the CRDAO

The initial membership of the DAO will be members of the CRDAO working group. Any active community member can join the DAO, the decision for the initial cohort will be approved by the AGP authors. Afterwards, new members will be voted in by the existing DAO members. High preference will be given to Flock and Nest team members, however, it is intended that community members external to the official Aragon teams are also included to create a diversity of perspectives.

#### Application for membership

- Members of the broader community can apply provided they can identify themselves as active community members in the forum, rocket chat, or GitHub. Recognizing community contributions requires being active in the community, there for a engagement is just as important as seniority.

#### Requirements

- Be an active community member

- Members of the DAO must vote in every round. This is a weekly commitment. If a member is not voting and cannot justify why they will be removed.

#### Onboarding

- For the first membership round, the AGP authors will be in charge of accepting/rejecting applications. After that, CRDAO members will manage membership autonomously. Once a CRDAO member, any member can create a vote to invite another participant to the DAO. Existing members must vote to approve that new member.

#### Offboarding

- If a member is not actively participating in weekly voting or discussions they will be removed.

### Intentions

The CRDAO and its members should be as autonomous as possible in the allocation of rewards. In the short term, DAO members will have equal voting weight over the allocations app. When SourceCred is ready to be implemented on Aragon voting weight will be based on reputation credits.

Any member of the Aragon community can nominate another member to be included in the next round of awards by adding mentioning them in the CRDAO forum thread. This will include who is being nominated and what they have done that is awesome and contributes to the community.

All legitimate nominations will be included in the weekly dot vote by the CRDAO community (or most likely a CRDAO admin responsible for upgrading the Address Book). In the case of a sybil attack or spammy nominations the CRDAO reserves the right to preclude nominees from any given dot vote. A nomination, and an addition to the weekly dot vote does not guarantee rewards. CRDAO members are free to vote however they want and can allocate all of their votes to one nominee or spread their voting weight between multiple nominees.

## Summing up

- The DAO will have a council of members. These members have two responsibilities
		- to vote to add or remove members to the CRDAO
		- to vote to allocate community rewards on a weekly basis

- Any Aragon community member should be able to nominate another Aragon community member for a reward

- The CRDAO plans to move towards being a truly decentralized autonomous organization
	- first by broadening the DAO's membership
	- then by using the SourceCred algorithm to give voting weight to all Aragon forum community members
